### PR TITLE
squid:S2039 - Member variable visibility should be specified

### PR DESCRIPTION
**Pull request by m-ezzat**
This pull request is focused on resolving occurrences of Sonar rule squid:S2039 - Member variable visibility should be specified

You can find more information about the issue here: https://dev.eclipse.org/sonar/coding_rules#q=squid:S2039

Please let me know if you have any questions.

M-Ezzat

### CODE REVIEW

File: src/main/java/hr/irb/fastRandomForest/Benchmark.java [hunk 1]
```diff
@@ -256,7 +256,7 @@ public static void main(String[] args) throws Exception {
         else
           result.add(new File(line + myExt));
       }
-
+      bufRdr.close();
```
Comment (ctrueden) [inline src/main/java/hr/irb/fastRandomForest/Benchmark.java]: This change, while likely a good one, does not match the stated commit message.

**Comment by ctrueden** [comment]
@iarganda I think it LGTM. I made one nitpicky comment, but in the interest of pragmatism and expediency, I recommend merging this as is.

File: src/main/java/hr/irb/fastRandomForest/Benchmark.java (hunk 1 above)
Comment (m-ezzat) [inline src/main/java/hr/irb/fastRandomForest/Benchmark.java]: yea, that's true, and I've reverted it in another commit.

**Comment by ctrueden** [comment]
@m-ezzat For Fiji projects, we prefer to keep a clean history. So it would be better to just squash these commits together and then force push. The PR will update.

http://imagej.net/How_to_contribute_to_an_existing_plugin_or_library#Modifying_your_commits

**Comment by m-ezzat** [comment]
yea, I prefer that as well, but I didn't want you to review again if there's any other requests in this PR, therefore, I know that squashing is better at the end before merging, especially in this big PR.

**Pull request merged**